gherkin/ruby/lib/gherkin/gherkin.rb

```diff
@@ -0,0 +1,23 @@
+require 'open3'
+require 'gherkin/protobuf_cucumber_messages'
+require 'gherkin/exe_file'
+
+module Gherkin
+  class Gherkin
+    def initialize(paths, print_source, print_ast, print_pickles)
+      @paths, @print_source, @print_ast, @print_pickles = paths, print_source, print_ast, print_pickles
+      path_pattern = "#{File.dirname(__FILE__)}/../../gherkin-go/gherkin-{{.OS}}-{{.Arch}}{{.Ext}}"
```

Comment:
I think this is causing troubles. If I leave with `/../../../` it doesn't pass locally, but now it's failing in Travis.

Reply:
My guess is it’s failing because you disabled the download of the executables.